.envrc

```diff
@@ -0,0 +1,15 @@
+strict_env
+
+if ! find_up .venv/pyvenv.cfg; then 
+  python3 -m venv .venv
+fi
+
+source ./.venv/bin/activate
+
+if ! find_up .venv/check.txt; then
+  # pip install .
+  pip install '.[dev]'
+  touch .venv/check.txt
+fi
```

Comment:
Use the `[ ]` notation for an `if` and also call the file something else, I think `.direnv` would be nice.

Reply:
Also, why is `pip install .` commented?